src/main/java/com/webperside/courseerpbackend/services/user/settings/UserSettingsBusinessServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.webperside.courseerpbackend.services.user.settings;
+
+import com.webperside.courseerpbackend.constants.UserConfigConstants;
+import com.webperside.courseerpbackend.exception.BaseException;
+import com.webperside.courseerpbackend.models.mybatis.language.Language;
+import com.webperside.courseerpbackend.repository.LanguageRepository;
+import com.webperside.courseerpbackend.repository.UserConfigRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class UserSettingsBusinessServiceImpl implements UserSettingsBusinessService{
+
+    private final UserConfigRepository userConfigRepository;
```

Comment:
I saw it now, 
So you have to use the related service instead of the repository.
And also there is a little trick here. so the flow will be like that, 

create a method inside UserConfigService named updateUserLanguage(Long userId, Long langId);
and also create a method (remove the current one, or rewrite it) inside UserConfigRepository named updateConfig(String key, Object value, Long userId)
and in the service's method, call the repository's method with related parameters
repo.updateConfig(DEFAULT_LANGUAGE, langId, userId);